ports/bddisasm/vcpkg.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "bddisasm",
+  "version-string": "1.31.8",
```

```suggestion
  "version": "1.32.1",
```
The latest release should be 1.32.1. For field name, please refer to https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md